python/ray/dag/tests/experimental/test_torch_tensor_dag.py

```diff
@@ -1218,14 +1201,7 @@ def test_torch_tensor_nccl_all_reduce_duplicate_actors(ray_start_regular):
     Test an error is thrown when two input nodes from the same actor bind to
     an all-reduce.
     """
-    if not USE_GPU:
-        pytest.skip("NCCL tests require GPUs")
-
-    assert (
-        sum(node["Resources"].get("GPU", 0) for node in ray.nodes()) > 1
-    ), "This test requires at least 2 GPUs"
-
-    actor_cls = TorchTensorWorker.options(num_cpus=0, num_gpus=1)
+    actor_cls = TorchTensorWorker.options(num_gpus=0)
```

Comment:
Does it work without `num_gpus=0`?

Reply:
Yes, it works without `num_gpus=0`. Stephanie commented that this test is only testing for errors and should not require GPU, so I changed it.

Reply:
Remove `num_gpus=0`.